src/transformers/models/roberta/modeling_roberta.py

```diff
@@ -864,7 +874,8 @@ def forward(
     """RoBERTa Model with a `language modeling` head on top for CLM fine-tuning. """, ROBERTA_START_DOCSTRING
 )
 class RobertaForCausalLM(RobertaPreTrainedModel):
-    _keys_to_ignore_on_load_missing = [r"position_ids", r"lm_head.decoder.bias"]
+    _keys_to_ignore_on_save = [r"position_ids", r"lm_head.decoder.weight", r"lm_head.decoder.bias"]
```

Comment:
I'm not sure if I need to add r"position_ids" here, at least 2 tests are failing because of that.

```

FAILED tests/test_modeling_roberta.py::RobertaModelTest::test_save_load__keys_to_ignore_on_save - AssertionError: 'position_ids' not found in Orde...
FAILED tests/test_modeling_roberta.py::RobertaModelIntegrationTest::test_lm_head_ignore_keys - AssertionError: Lists differ: ['position_ids', 'lm_...
```